Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ruby:3.1.2-slim
```

Comment:
Is this version of Ruby accurate, knowing that we need a version < 3 to run the game?